app/controllers/bananas2_controller.rb

```diff
@@ -66,13 +63,14 @@ def destroy
   end
 
   private
-    # Use callbacks to share common setup or constraints between actions.
-    def set_banana
-      @banana = Banana.find(params[:id])
-    end
 
-    # Never trust parameters from the scary internet, only allow the white list through.
-    def banana_params
-      params.require(:banana).permit(:name, :price)
-    end
+  # Use callbacks to share common setup or constraints between actions.
+  def set_banana
+    @banana = Banana.find(params[:id])
+  end
+
+  # Never trust parameters from the scary internet, only allow the white list through.
```

Comment:
__[rubocop]__
* Line is too long. [86/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)